### DISCUSSION TITLE
Berksey/about-yukifuri-merikuri/

### DISCUSSION THREAD

**Discussion by giscus** [NachtgeistW/Berksey#58] [General]
# Berksey/about-yukifuri-merikuri/

讲点有的没的创作故事。

https://nachtgeistw.github.io/Berksey/about-yukifuri-merikuri/

**Comment by Chlorie** [comment]
由此可见夜轮是一个非常优秀的人，无论是她的个性还是她的才能都非常值得称赞（以下略去好几百字

**Reply by NachtgeistW** [reply]
你他喵的草
